app/controllers/reactions_controller.rb

```diff
@@ -120,7 +120,9 @@ def build_reaction(category)
       reactable_type: params[:reactable_type],
       category: category
     }
-    create_params[:status] = "confirmed" if current_user&.any_admin?
+    if NEGATIVE_CATEGORIES.include?(category) && current_user&.any_admin?
```

Comment:
Should we maybe switch the two conditions around so we exit as early as possible (no current user or they are not an admin) before doing the array inclusion check? It's not particularly important in this case as the latter check is cheap, but maybe a good habit to get into.

Reply:
Good call, done!